Carmunication/Views/MyPage/FriendAddView.swift

```diff
@@ -0,0 +1,219 @@
+        clearButton.setImage(UIImage(systemName: "xmark.circle.fill"), for: .normal)
+        clearButton.tintColor = UIColor.semantic.accPrimary
+        return clearButton
+    }()
+
+    // MARK: - 텍스트 필드 검색 버튼
+    lazy var friendSearchButton: UIButton = {
+        let friendSearchButton = UIButton()
+        friendSearchButton.setTitle("검색", for: .normal)
+        friendSearchButton.setTitleColor(UIColor.semantic.accPrimary, for: .normal)
+        friendSearchButton.titleLabel?.font = UIFont.carmuFont.body2Long
+        return friendSearchButton
+    }()
+
+    // MARK: - 검색된 친구 박스
+    lazy var searchedFriendView: UIView = {
+        let searchedFriendView = UIView()
+        searchedFriendView.backgroundColor = UIColor.semantic.backgroundAddress
+        searchedFriendView.layer.cornerRadius = 16
+        return searchedFriendView
+    }()
+
+    // 친구 검색 결과 없음 라벨
+    lazy var noResultLabel: UILabel = {
+        let noResultLabel = UILabel()
+        noResultLabel.text = "검색된 친구가 없습니다."
+        noResultLabel.font = UIFont.carmuFont.body1
+        noResultLabel.textColor = UIColor.semantic.textPrimary
+        return noResultLabel
+    }()
+
+    // TODO: - 검색된 친구 셀 보여주기 구현 필요
+
+    // MARK: - 친구 추가하기 버튼
+    lazy var friendAddButton: UIButton = {
+        let friendAddButton = UIButton()
+        friendAddButton.backgroundColor = UIColor.theme.blue6
+        friendAddButton.setTitle("친구 추가하기", for: .normal)
+        friendAddButton.titleLabel?.font = UIFont.carmuFont.subhead3
+        friendAddButton.setTitleColor(UIColor.theme.white, for: .normal)
+        friendAddButton.layer.cornerRadius = 30
+
+        friendAddButton.setTitleColor(UIColor.semantic.textDisableBT, for: .disabled)
+        return friendAddButton
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override func draw(_ rect: CGRect) {
+        setupViews()
+        setAutoLayout()
+    }
+
+    func setupViews() {
+        addSubview(headerBar)
+        addSubview(friendSearchTextFieldView)
+        addSubview(searchedFriendView)
+        addSubview(friendAddButton)
+
+        headerBar.addSubview(headerTitleLabel)
+        headerBar.addSubview(closeButton)
+
+        friendSearchTextFieldView.addSubview(friendSearchTextField)
+
+        textFieldUtilityStackView.addArrangedSubview(clearButton)
+        textFieldUtilityStackView.addArrangedSubview(friendSearchButton)
+
+        searchedFriendView.addSubview(noResultLabel)
+    }
+
+    // MARK: - 오토 레이아웃 설정 메서드
+    func setAutoLayout() {
+        headerBar.snp.makeConstraints { make in
+            make.top.equalToSuperview().inset(16)
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(42)
+        }
+        headerTitleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.centerY.equalToSuperview()
```

Comment:
요 메서드는 한 줄로 작성할 수 있을 것 같아요 😊

```suggestion
            make.center.equalToSuperview()
```

Reply:
> 요 메서드는 한 줄로 작성할 수 있을 것 같아요 😊

오 이건 저도 몰랐는데. 꿀팁이군요 !